src/common/components/map/controls/Controls.module.scss

```diff
@@ -16,25 +16,18 @@
   border: 1px solid black;
 }
 
-.controlMenu {
-  background: var(--color-white);
-  padding: var(--spacing-s) var(--spacing-xs) var(--spacing-xs) var(--spacing-3-xs);
-  border: 2px solid var(--color-black-60) !important;
-
-  &__divider {
-    margin-top: var(--spacing-xs);
-    margin-bottom: var(--spacing-xs);
-  }
-}
-
 .tileLayerControl {
   position: absolute;
-  top: var(--spacing-s);
-  right: var(--spacing-s);
+  top: var(--spacing-xs);
+  right: 9px;
   background: var(--color-black-80);
 
+  &:hover {
+    outline: 1px solid black;
```

Comment:
Border colors are of different shade in layers and fullscreen buttons, which looks a bit odd. Hover effect is also a bit difficult to see in Ortokartta mode. One option would be to use same hover style as in drawing tools for consistency.

Reply:
Good point. Changed it to use the same style as in drawing tools.